aiidalab_widgets_base/elns.py

```diff
@@ -261,7 +279,14 @@ def check_connection(self, _=None):
 
     def display_eln_config(self, value=None):
         """Display ELN configuration specific to the selected type of ELN."""
-        eln_class = get_eln_connector(self.eln_types.value)
+        try:
+            eln_class = get_eln_connector(self.eln_types.value)
+        except NotImplementedError as err:
+            with self._output:
+                clear_output()
+                display(ipw.HTML("&#10060;" + str(err)))
```

Comment:
Do we want to show a more user-friendly text also in this case? Something like "Unexpected error. There is no connector implemented for the ELN type you choose. Report his bug at ...."

Reply:
> Might be actually a candidate for https://github.com/aiidalab/aiidalab-widgets-base/blob/develop/aiidalab_widgets_base/bug_report.py?

I would say this is mostly for the simulation apps where there can be quite unexpected errors. Here the possible errors are quite limited, so I would prefer to handle them as explicitly as possible.